pillarbox-core-business/src/main/java/ch/srgssr/pillarbox/core/business/MediaItemUrn.kt

```diff
@@ -28,6 +28,7 @@ object MediaItemUrn {
                 MediaMetadata.Builder()
                     .setTitle(title)
                     .setSubtitle(subtitle)
+                    // TODO Integrate `ImageScaleService` once we know if this class will be used or not
```

Comment:
Is this a TODO for the future or has it been addressed (or not)?

Reply:
This class (`MediaItemUrn`) was created in September, but is currently not used.
I'd like to check its status with @StaehliJ, to know if it makes sense to update it, or if we should delete, or if the change is not necessary here.